pkg/gh/github.go

```diff
@@ -86,6 +63,41 @@ func (o GitOptions) handleRateLimit(action func() (*github.Response, error)) err
 	return nil
 }
 
+func (o GitOptions) handleRateLimitList(action func(opt *github.ListOptions) (*github.Response, error)) error {
+	opt := &github.ListOptions{
+		PerPage: 30,
+	}
+
+	for {
+		resp, err := action(opt)
```

Comment:
`err` check

Reply:
The err check is being done after the unauthorised status check.

Reply:
I've reworked the error handling to be clearer so think this is better now